### ISSUE TITLE
touch on q-drawer calls error

### ISSUE THREAD

**Issue by matro-skin** [quasarframework/quasar#6125]
**Describe the bug**
While clicking any place when drawer is opened - both the drawer either backdrop, I see the error in console.

Click on drawer calls:
```
Uncaught TypeError: Cannot read property 'event' of undefined
    at shouldStart (touch.js:114)
    at HTMLElement.touchStart (TouchPan.js:148)
```

Click on backdrop calls:
```
Uncaught TypeError: Cannot read property 'event' of undefined
    at shouldStart (touch.js:114)
    at HTMLDivElement.touchStart (TouchPan.js:148)
```

**To Reproduce**
1. Open `q-drawer`
2. Click any place

**Platform (please complete the following information):**
OS: Win10 x64
Node: v12.13.0
NPM: 6.13.6
Browsers: Chrome 79.0.3945.117 (Official Build) (64-bit)

**Comment by matro-skin** [comment]
Error only in Chrome,
IE, FF are ok.

**Comment by rstoenescu** [comment]
Fixed already in future v1.7.4 (to be released in half an hour).
